ports/aom/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "aom",
   "version-semver": "3.2.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "AV1 codec library",
   "homepage": "https://aomedia.googlesource.com/aom",
-  "supports": "!uwp & !arm",
+  "supports": "!(windows & (arm | arm64))",
```

```suggestion
  "supports": "!(windows & arm & uwp)",
```
According to your comment:
> Add support for building libavif/aom/libyuv on ARM and UWP. The only configurations that do not build are arm-windows and arm64-windows. (arm-uwp and arm64-uwp builds)

This condition should be `!(windows & arm & uwp)`?

---

hmmm I thought `windows` include `uwp`? then (windows & arm & uwp) should be uwp-arm64/uwp-arm? I think non-uwp windows is also not supported.

---

`windows` contains `uwp` and `non-uwp`, so it may be `!(arm & uwp)`?
In the past, this port absoultly support non-Windows.

---

I tested, this port builds with arm64-uwp too, guess I will just remove the `support` entry. libavif also builds on arm64-uwp fine

---

hmm, looks like the non-uwp windows arm64 will fail...

---

Let's wait for all tests finish.